### ISSUE TITLE
Sending cart captions in non-English

### ISSUE THREAD

**Issue by GauravES** [MicrosoftDocs/msteams-docs#8530]
The CARTCaption  API for Teams Meetings suggest that a lang code can be passed in the POST.
However even though the product seems to accept a different langcode as a param ( say hi-IN) without error, the captions do not appear on the Teams client.   In the Teams options, there are language settings for live captions, but the CART language option appears to only support en-US.

**Is it possible to send CART captions in a different language using the CART API?**

Sample provided on this page:

POST /cartcaption?meetingid=04751eac-30e6-47d9-9c3f-0b4ebe8e30d9&token=04751eac&**lang=en-us** HTTP/1.1
Host: api.captions.office.microsoft.com
Content-Type: text/plain
Content-Length: 22
Hello I’m Cortana, welcome to my meeting.

Changing the lang code to hi-IN and sending UTF-8 characters in that language is accepted by the server, but does not shown in Teams client.   On client, it seems there is no option to select a different CART language. 

![image](https://user-images.githubusercontent.com/31090404/234443389-16e3f58b-8872-4092-a4cd-cfeef3026666.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ac148a35-2743-e16e-2b49-3ead0a1ecebc
* Version Independent ID: ac148a35-2743-e16e-2b49-3ead0a1ecebc
* Content: [Meeting apps APIs - Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?tabs=dotnet)
* Content Source: [msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md)
* Service: **msteams**
* GitHub Login: @v-sdhakshina
* Microsoft Alias: **lajanuar**

**Comment by Nivedipa-MSFT** [comment]
@GauravES - Thanks for reporting your issue. We will test this and update you soon.

**Comment by Nivedipa-MSFT** [comment]
@GauravES - We have tested the cart captions API with lang parameter but its not changing CART language in Teams Client.
We have tested this on Teams Client version 1.6.00.10677. and Postman version v10.13.4

I have set CART language to Arabic and using API trying to change it to en-US but it's not changing the language in Teams client.


![image](https://user-images.githubusercontent.com/92150070/234838582-23de0cd3-c36d-4555-84cc-ebe24f273dfe.png)


![image](https://user-images.githubusercontent.com/92150070/234837108-2fbfff18-d055-4d10-a3f2-ccf8daaff1d2.png)

We have raised a bug for the same. We will let you know once we get any updates on it.
